### ISSUE TITLE
memleak on RAI_ModelSerializeORT RAI_ONNXBuffer* onnxbuffer = (RAI_ONNXBuffer*)model->data;

### ISSUE THREAD

**Issue by filipecosta90** [RedisAI/RedisAI#293]
leak on 
``` c
RAI_ONNXBuffer* onnxbuffer = (RAI_ONNXBuffer*)model->data; 
```

```
==28618== 26,454 bytes in 1 blocks are definitely lost in loss record 132,301 of 132,310
==28618==    at 0x4C31B25: calloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==28618==    by 0x144DB6: zcalloc (zmalloc.c:131)
==28618==    by 0x1C6FAC: RM_Calloc (module.c:335)
==28618==    by 0x236DCFF6: RAI_ModelSerializeORT (onnxruntime.c:508)
==28618==    by 0x67E7B20: RAI_ModelSerialize (model.c:403)
==28618==    by 0x67D823A: RedisAI_ModelGet_RedisCommand (redisai.c:717)
==28618==    by 0x1C76BE: RedisModuleCommandDispatcher (module.c:575)
==28618==    by 0x13D315: call (server.c:2468)
==28618==    by 0x13DFCA: processCommand (server.c:2763)
==28618==    by 0x14F086: processInputBuffer (networking.c:1470)
==28618==    by 0x14F1DB: processInputBufferAndReplicate (networking.c:1505)
==28618==    by 0x14F551: readQueryFromClient (networking.c:1587)
```

**Comment by lantiga** [comment]
Solved in #289 (see 99b82c1)

**Comment by filipecosta90** [comment]
confirming that leak was solved
